[libwebsockets] Build failed

Hi, not sure if this is a bug or if I'm using registries incorrectly -- first time user!

**Describe the bug**
I believe the bug is because of a package I pull in as a registry package that has the same name of a package outside of my registry.

For example, my manifest file specifies "mosquitto", and "pthreads" which is my own version of the "pthreads" vcpkg package which I keep in a private GH enterprise instance. When mosquitto is installing its dependencies, it also lists "pthreads" (the standard one vcpkg has listed in its repository) as a dependency, it pulls in my private package from the registry instead of pulling in from vcpkg's list.

There are no issues with the installation if I don't specify my registry.

**Environment**
- OS: Windows 10
- Compiler: MSVC 19.28.29912.0

**To Reproduce**
Steps to reproduce the behavior:
1. Create vcpkg.json:
```
{
    "name": "test",
    "version": "1.0.0",
    "dependencies": [ "mosquitto", { "name": "pthreads", "platform": "windows" } ],
    "overrides": [
        { "name": "mosquitto", "version": "1.6.8", "port-version": 3 }
    ],
    "builtin-baseline": "030cfaa24de9ea1bbf0a4d9c615ce7312ba77af1"
}
```
2. Create vcpkg-configuration.json:
```
{
    "registries": [
        {
            "kind": "git",
            "repository": "https://<URL to private GitHub Enterprise instance>/vcpkg-registry",
            "packages": [ "pthreads" ]
        }
    ]
}
```
3. `./vcpkg install --feature-flags="manifests,versions,registries"`


**Expected behavior**
When vcpkg packages pull in dependencies, it pulls them in from vcpkg's list.

**Failure logs** 
`vcpkg install` output:
```
The following packages will be built and installed:
    c-ares[core]:x86-windows -> 1.17.1 -- C:\Users\acalandra\test\vcpkg\buildtrees\versioning\versions\c-ares\a35b0a23f91ec79cecdcb83c82840f7305dbd709
    libuv[core]:x86-windows -> 1.39.0 -- C:\Users\acalandra\test\vcpkg\buildtrees\versioning\versions\libuv\18afc7f0e81eb4d9c8a9d8d1f4c15a96cb4f2db1
    libwebsockets[core]:x86-windows -> 4.1.6 -- C:\Users\acalandra\test\vcpkg\buildtrees\versioning\versions\libwebsockets\d980cd5098de3cfba2ecd8e63490cf1013545dfd
    mosquitto[core]:x86-windows -> 1.6.8#3 -- C:\Users\acalandra\test\vcpkg\buildtrees\versioning\versions\mosquitto\e50ddad9108597e09097e1853e116a62d650044d
    openssl[core]:x86-windows -> 1.1.1j -- C:\Users\acalandra\test\vcpkg\buildtrees\versioning\versions\openssl\009a7ee95741b939859c77b8e1ddd6c146ffe54e
    pthreads[core]:x86-windows -> 1.0.0 -- C:\Users\acalandra\AppData\Local\vcpkg\registries\git-trees\bb4c600c046c3717bc9fa888844247495eda8a2d
    zlib[core]:x86-windows -> 1.2.11#9 -- C:\Users\acalandra\test\vcpkg\buildtrees\versioning\versions\zlib\827111046e37c98153d9d82bb6fa4183b6d728e4
Could not locate cached archive: C:\Users\acalandra\AppData\Local\vcpkg\archives\df\df65ec229cd0271799a8de9aa8a1e946335c4be9.zip
Using cached binary package: C:\Users\acalandra\AppData\Local\vcpkg\archives\a7\a77c7fbbfbd0204c7b65d03e4066e122be8e69a5.zip
Using cached binary package: C:\Users\acalandra\AppData\Local\vcpkg\archives\6c\6c5020be3e9f566a95250c69a8d661e737c017c4.zip
Using cached binary package: C:\Users\acalandra\AppData\Local\vcpkg\archives\a4\a4a32af0bdf1db916321aed0add13d3d1dacddcb.zip
Could not locate cached archive: C:\Users\acalandra\AppData\Local\vcpkg\archives\9f\9f44457ae1a17fcc69c7133c516c6e33316a7b4d.zip
Using cached binary package: C:\Users\acalandra\AppData\Local\vcpkg\archives\46\46945da58423b348383ae036ff9b25565aed8a0d.zip
Could not locate cached archive: C:\Users\acalandra\AppData\Local\vcpkg\archives\7f\7fa470671d9ebc8bdb3ab5b69e249be4435e7e4c.zip
Starting package 1/7: pthreads:x86-windows
Building package pthreads[core]:x86-windows...
-- Installing port from location: C:\Users\acalandra\AppData\Local\vcpkg\registries\git-trees\bb4c600c046c3717bc9fa888844247495eda8a2d
-- Fetching https://<URL of private GitHub Enterprise server>/pthreads.git...
-- Extracting source C:/Users/acalandra/test/vcpkg/downloads/pthreads-2b27fe212d2ea17203bcf824c13c60b0753b2729.tar.gz
-- Using source at C:/Users/acalandra/test/vcpkg/buildtrees/pthreads/src/b0753b2729-917ff5b6ad.clean
-- Performing post-build validation
-- Performing post-build validation done
Stored binary cache: C:\Users\acalandra\AppData\Local\vcpkg\archives\df\df65ec229cd0271799a8de9aa8a1e946335c4be9.zip
Building package pthreads[core]:x86-windows... done
Installing package pthreads[core]:x86-windows...
Installing package pthreads[core]:x86-windows... done
Elapsed time for package pthreads:x86-windows: 5.342 s
Starting package 2/7: openssl:x86-windows
Building package openssl[core]:x86-windows...
Building package openssl[core]:x86-windows... done
Installing package openssl[core]:x86-windows...
Installing package openssl[core]:x86-windows... done
Elapsed time for package openssl:x86-windows: 225 ms
Starting package 3/7: zlib:x86-windows
Building package zlib[core]:x86-windows...
Building package zlib[core]:x86-windows... done
Installing package zlib[core]:x86-windows...
Installing package zlib[core]:x86-windows... done
Elapsed time for package zlib:x86-windows: 26.89 ms
Starting package 4/7: libuv:x86-windows
Building package libuv[core]:x86-windows...
Building package libuv[core]:x86-windows... done
Installing package libuv[core]:x86-windows...
Installing package libuv[core]:x86-windows... done
Elapsed time for package libuv:x86-windows: 47.07 ms
Starting package 5/7: libwebsockets:x86-windows
Building package libwebsockets[core]:x86-windows...
-- Installing port from location: C:\Users\acalandra\test\vcpkg\buildtrees\versioning\versions\libwebsockets\d980cd5098de3cfba2ecd8e63490cf1013545dfd
-- Downloading https://github.com/warmcat/libwebsockets/archive/736f0739ed8c75db0e29c7678b5a9194e957c347.tar.gz -> warmcat-libwebsockets-736f0739ed8c75db0e29c7678b5a9194e957c347.tar.gz...
-- Extracting source C:/Users/acalandra/test/vcpkg/downloads/warmcat-libwebsockets-736f0739ed8c75db0e29c7678b5a9194e957c347.tar.gz
-- Applying patch CMakeLists.patch
-- Applying patch fix-build-error.patch
-- Using source at C:/Users/acalandra/test/vcpkg/buildtrees/libwebsockets/src/94e957c347-2297ce2a93.clean
-- Found external ninja('1.10.2').
-- Configuring x86-windows
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:105 (message):
    Command failed: ninja -v
    Working Directory: C:/Users/acalandra/test/vcpkg/buildtrees/libwebsockets/x86-windows-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      C:\Users\acalandra\test\vcpkg\buildtrees\libwebsockets\config-x86-windows-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_cmake.cmake:324 (vcpkg_execute_required_process)
  buildtrees/versioning/versions/libwebsockets/d980cd5098de3cfba2ecd8e63490cf1013545dfd/portfile.cmake:134 (vcpkg_configure_cmake)
  scripts/ports.cmake:142 (include)


Error: Building package libwebsockets:x86-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: libwebsockets:x86-windows
  Vcpkg version: 2021-01-13-d67989bce1043b98092ac45996a8230a059a2d7e
```

Log from `C:\Users\acalandra\test\vcpkg\buildtrees\libwebsockets\config-x86-windows-out.log`:
```
[1/2] cmd /c "cd ../../x86-windows-dbg && "C:/Users/acalandra/test/vcpkg/downloads/tools/cmake-3.19.2-windows/cmake-3.19.2-win32-x86/bin/cmake.exe" "C:/Users/acalandra/test/vcpkg/buildtrees/libwebsockets/src/94e957c347-2297ce2a93.clean" "-DLWS_WITH_STATIC=0" "-DLWS_WITH_SHARED=1" "-DLWS_WITH_GENCRYPTO=ON" "-DLWS_WITH_TLS=ON" "-DLWS_WITH_BUNDLED_ZLIB=OFF" "-DLWS_WITHOUT_TESTAPPS=ON" "-DLWS_IPV6=ON" "-DLWS_WITH_HTTP2=ON" "-DLWS_WITH_HTTP_STREAM_COMPRESSION=ON" "-DLWS_WITH_LIBUV=ON" "-DCMAKE_MAKE_PROGRAM=C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Users/acalandra/test/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x86-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/Users/acalandra/test/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x86" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/Users/acalandra/test/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/Users/acalandra/test/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=C:/Users/acalandra/test/vcpkg/packages/libwebsockets_x86-windows/debug""
FAILED: ../../x86-windows-dbg/CMakeCache.txt 
cmd /c "cd ../../x86-windows-dbg && "C:/Users/acalandra/test/vcpkg/downloads/tools/cmake-3.19.2-windows/cmake-3.19.2-win32-x86/bin/cmake.exe" "C:/Users/acalandra/test/vcpkg/buildtrees/libwebsockets/src/94e957c347-2297ce2a93.clean" "-DLWS_WITH_STATIC=0" "-DLWS_WITH_SHARED=1" "-DLWS_WITH_GENCRYPTO=ON" "-DLWS_WITH_TLS=ON" "-DLWS_WITH_BUNDLED_ZLIB=OFF" "-DLWS_WITHOUT_TESTAPPS=ON" "-DLWS_IPV6=ON" "-DLWS_WITH_HTTP2=ON" "-DLWS_WITH_HTTP_STREAM_COMPRESSION=ON" "-DLWS_WITH_LIBUV=ON" "-DCMAKE_MAKE_PROGRAM=C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Users/acalandra/test/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x86-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/Users/acalandra/test/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x86" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/Users/acalandra/test/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/Users/acalandra/test/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=C:/Users/acalandra/test/vcpkg/packages/libwebsockets_x86-windows/debug""
-- The C compiler identification is MSVC 19.28.29912.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/VC/Tools/MSVC/14.28.29910/bin/Hostx64/x86/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Compiled with LWS_WITH_DIR and LWS_WITH_LEJP_CONF
SMD requires pthreads
-- Found Git: C:/Program Files/Git/cmd/git.exe (found version "2.28.0.windows.1") 
Git commit hash: 2020.11-1-1039-gfe112d830
-- CMAKE_TOOLCHAIN_FILE='C:/Users/acalandra/test/vcpkg/scripts/buildsystems/vcpkg.cmake'
CMake Error at C:/Users/acalandra/test/vcpkg/scripts/buildsystems/vcpkg.cmake:861 (_find_package):
  By not providing "Findpthreads_windows.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "pthreads_windows", but CMake did not find one.

  Could not find a package configuration file provided by "pthreads_windows"
  with any of the following names:

    pthreads_windowsConfig.cmake
    pthreads_windows-config.cmake

  Add the installation prefix of "pthreads_windows" to CMAKE_PREFIX_PATH or
  set "pthreads_windows_DIR" to a directory containing one of the above
  files.  If "pthreads_windows" provides a separate development package or
  SDK, be sure it has been installed.
Call Stack (most recent call first):
  CMakeLists.txt:426 (find_package)


-- Configuring incomplete, errors occurred!
See also "C:/Users/acalandra/test/vcpkg/buildtrees/libwebsockets/x86-windows-dbg/CMakeFiles/CMakeOutput.log".
[2/2] cmd /c "cd .. && "C:/Users/acalandra/test/vcpkg/downloads/tools/cmake-3.19.2-windows/cmake-3.19.2-win32-x86/bin/cmake.exe" "C:/Users/acalandra/test/vcpkg/buildtrees/libwebsockets/src/94e957c347-2297ce2a93.clean" "-DLWS_WITH_STATIC=0" "-DLWS_WITH_SHARED=1" "-DLWS_WITH_GENCRYPTO=ON" "-DLWS_WITH_TLS=ON" "-DLWS_WITH_BUNDLED_ZLIB=OFF" "-DLWS_WITHOUT_TESTAPPS=ON" "-DLWS_IPV6=ON" "-DLWS_WITH_HTTP2=ON" "-DLWS_WITH_HTTP_STREAM_COMPRESSION=ON" "-DLWS_WITH_LIBUV=ON" "-DCMAKE_MAKE_PROGRAM=C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Users/acalandra/test/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x86-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/Users/acalandra/test/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x86" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/Users/acalandra/test/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/Users/acalandra/test/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/Users/acalandra/test/vcpkg/packages/libwebsockets_x86-windows""
FAILED: ../CMakeCache.txt 
cmd /c "cd .. && "C:/Users/acalandra/test/vcpkg/downloads/tools/cmake-3.19.2-windows/cmake-3.19.2-win32-x86/bin/cmake.exe" "C:/Users/acalandra/test/vcpkg/buildtrees/libwebsockets/src/94e957c347-2297ce2a93.clean" "-DLWS_WITH_STATIC=0" "-DLWS_WITH_SHARED=1" "-DLWS_WITH_GENCRYPTO=ON" "-DLWS_WITH_TLS=ON" "-DLWS_WITH_BUNDLED_ZLIB=OFF" "-DLWS_WITHOUT_TESTAPPS=ON" "-DLWS_IPV6=ON" "-DLWS_WITH_HTTP2=ON" "-DLWS_WITH_HTTP_STREAM_COMPRESSION=ON" "-DLWS_WITH_LIBUV=ON" "-DCMAKE_MAKE_PROGRAM=C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/Users/acalandra/test/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x86-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v142" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/Users/acalandra/test/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x86" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/Users/acalandra/test/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/Users/acalandra/test/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/Users/acalandra/test/vcpkg/packages/libwebsockets_x86-windows""
-- The C compiler identification is MSVC 19.28.29912.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/VC/Tools/MSVC/14.28.29910/bin/Hostx64/x86/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Compiled with LWS_WITH_DIR and LWS_WITH_LEJP_CONF
SMD requires pthreads
-- Found Git: C:/Program Files/Git/cmd/git.exe (found version "2.28.0.windows.1") 
Git commit hash: 2020.11-1-1039-gfe112d830
-- CMAKE_TOOLCHAIN_FILE='C:/Users/acalandra/test/vcpkg/scripts/buildsystems/vcpkg.cmake'
CMake Error at C:/Users/acalandra/test/vcpkg/scripts/buildsystems/vcpkg.cmake:861 (_find_package):
  By not providing "Findpthreads_windows.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "pthreads_windows", but CMake did not find one.

  Could not find a package configuration file provided by "pthreads_windows"
  with any of the following names:

    pthreads_windowsConfig.cmake
    pthreads_windows-config.cmake

  Add the installation prefix of "pthreads_windows" to CMAKE_PREFIX_PATH or
  set "pthreads_windows_DIR" to a directory containing one of the above
  files.  If "pthreads_windows" provides a separate development package or
  SDK, be sure it has been installed.
Call Stack (most recent call first):
  CMakeLists.txt:426 (find_package)


-- Configuring incomplete, errors occurred!
See also "C:/Users/acalandra/test/vcpkg/buildtrees/libwebsockets/x86-windows-rel/CMakeFiles/CMakeOutput.log".
ninja: build stopped: subcommand failed.
```

--- a/ports/libwebsockets/vcpkg.json
+++ b/ports/libwebsockets/vcpkg.json
@@ -1,17 +1,14 @@
 {
   "name": "libwebsockets",
   "version-semver": "4.1.6",
-  "port-version": 1,
+  "port-version": 2,
   "description": "Libwebsockets is a lightweight pure C library built to use minimal CPU and memory resources, and provide fast throughput in both directions as client or server.",
   "homepage": "https://github.com/warmcat/libwebsockets",
   "supports": "!(arm | uwp)",
   "dependencies": [
     "libuv",
     "openssl",
-    {
-      "name": "pthreads",
-      "platform": "!windows"
-    },
+    "pthreads",
     "zlib"
   ]
 }

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -3610,7 +3610,7 @@
     },
     "libwebsockets": {
       "baseline": "4.1.6",
-      "port-version": 1
+      "port-version": 2
     },
     "libxdiff": {
       "baseline": "0.23",

--- a/versions/l-/libwebsockets.json
+++ b/versions/l-/libwebsockets.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "fa509ae21580082ef8aea72e394bb7005f689371",
+      "version-semver": "4.1.6",
+      "port-version": 2
+    },
+    {
       "git-tree": "1846ac46a72330104a9a4c13042cdb19ae8a42c4",
       "version-semver": "4.1.6",
       "port-version": 1

